.devops/nix/package.nix

```diff
@@ -87,6 +88,11 @@ let
     ]
   );
 
+  darwinSymlinks = runCommand "darwin-build-symlinks" {} ''
```

Comment:
Maybe rename to `xcrunHost`/`hostsXcrun`. I also think that `makeWrapper` was a better solution but it's ok if you keep this as is